be/src/util/mem_info.cpp

```diff
@@ -34,39 +34,70 @@
 
 #include "util/mem_info.h"
 
+
 #include <cstdlib>
 #include <fstream>
 #include <iostream>
+#include <linux/magic.h>
+#include <sys/vfs.h>
 
 #include "fs/fs_util.h"
 #include "gutil/strings/split.h"
+#include "util/errno.h"
 #include "util/pretty_printer.h"
 #include "util/string_parser.hpp"
 
 namespace starrocks {
 
+#ifndef CGROUP2_SUPER_MAGIC
+#define CGROUP2_SUPER_MAGIC 0x63677270
```

Comment:
Should give some comments about this magic number.
And why not introduce this number by include some headers?

Reply:
The lower Linux kernel does not define this magic number. So I copy it from a high Linux version in `linux/magic.h`